taui/src/components/route-card.js

```diff
@@ -104,7 +104,7 @@ class RouteCard extends React.PureComponent<Props> {
                 </span>
                 {' '}
                 <span className='neighborhood-summary__location'>
-                  {currentDestination && t('TripPurpose.' + currentDestination.purpose).toLowerCase()}
+                  {currentDestination && t(currentDestination.purpose).toLowerCase()}
```

Comment:
Ditto here, if `TripPurpose` is kept out of `destination.purpose` then this can be left as it was.